### PR TITLE
Point source map links to Github

### LINKED ISSUE #275: Source map links are wrong
Use scala-js-dom with scalajs-bundler and you'll get a bunch of warnings about `/localhome/doeraene` not existing.

```
[info] WARNING in ./demo-fastopt.js
[info] Cannot find source file '../../../../../../../../localhome/doeraene/projects/scalajs-dom/src/main/scala/org/scalajs/dom/package.scala': Error: Can't resolve '../../../../../../../../localhome/doeraene/projects/scalajs-dom/src/main/scala/org/scalajs/dom/package.scala' in '/home/golly/projects/misc/target/scala-2.12/scalajs-bundler/main'
[info]  @ ./fastopt-launcher.js 1:0-91
```

Think you need something like [this](https://github.com/japgolly/scalajs-react/blob/v1.0.0-RC1/project/Build.scala#L74-L81).

### PR DESCRIPTION
**Pull request by japgolly**
Fixes #275

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -22,6 +22,12 @@ homepage := Some(url("http://scala-js.org/"))
 
 licenses += ("MIT", url("http://opensource.org/licenses/mit-license.php"))
 
+scalacOptions ++= (if (isSnapshot.value) Seq.empty else Seq({
+	val a = baseDirectory.value.toURI
+	val g = "https://raw.githubusercontent.com/scala-js/scala-js-dom"
+	s"-P:scalajs:mapSourceURI:$a->$g/v${version.value}/"
+}))
```
Comment (sjrd) [inline build.sbt]: Uh oh, evil tabs ;) Please use 2-space indentation inside `{}`. Also, I feel like `{}` would be more appropriate around the `if` than `()` in this case, given the length of the expression inside.

Comment (japgolly) [inline build.sbt]: 1. Oh god, there were tabs in there?! I feel shame.
2. Oh god, The Great Style Wall of @sjrd.

**Comment by japgolly** [comment]
Ok updated, try now

**Comment by sjrd** [comment]
Good :) Could you squash the two commits, please?

**Comment by japgolly** [comment]
👍

**Comment by japgolly** [comment]
Although FYI you can squash it yourself when you merge. The merge button has a dropdown with various merge options.

**Comment by sjrd** [comment]
> Although FYI you can squash it yourself when you merge. The merge button has a dropdown with various merge options.

I know, but if I do that I only get the squashed commit, instead of the squashed commit + a merge commit.

**Pull request merged**